NuGet.config

```diff
@@ -6,11 +6,8 @@
   <packageSources>
     <clear />
     <add key="nuget.org" value="https://api.nuget.org/v3/index.json" />
-    <add key="dotnet-core" value="https://dotnetfeed.blob.core.windows.net/dotnet-core/index.json" />
-    <add key="dotnet-fsharp" value="https://dotnet.myget.org/F/fsharp/api/v3/index.json" />
-    <add key="dotnet-try" value="https://dotnet.myget.org/F/dotnet-try/api/v3/index.json" />
```

Comment:
Both `dotnet-fsharp` and `dotnet-try` are required for this repo.

Reply:
I don't think that's the case. dotnet5 has the fsharp packages and does not appear to be getting regular updates anyway.